src/odemis/acq/orsay_milling.py

```diff
@@ -177,6 +180,17 @@ def do_milling(self):
             obj_to_mill.centerOfMassX = ((rect[2] + rect[0]) / 2 - 0.5) * fov
             obj_to_mill.centerOfMassY = (0.5 - (rect[3] + rect[1]) / 2) * fov
 
+            step_distance = probe_size * (1 - overlap)
```

Comment:
This is different for X & Y. `overlap` is a tuple.
```suggestion
            step_distance = (probe_size * (1 - overlap[0]),
                             probe_size * (1 - overlap[1]))
```
(And then update the line below.